extensions/wikia/AdEngine/js/provider/szymon.js

```diff
@@ -0,0 +1,33 @@
+/*global define*/
+/*jslint nomen: true*/
+/*jshint camelcase: false*/
+define('ext.wikia.adEngine.provider.szymon', [
+	'wikia.log',
+	'wikia.document'
+], function (log, document) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.provider.szymon';
+
+	function canHandleSlot(slot) {
+		log(['canHandleSlot', slot, slot === 'TOP_LEADERBOARD'], 'info', logGroup);
+		return slot === 'TOP_LEADERBOARD';
+	}
+
+	function fillInSlot(slotname, success) {
+		var slot = document.getElementById(slotname),
+			adTemplate = '<img src="http://lorempixel.com/728/90/" alt="{lorempixel}" />';
```

Comment:
I didn't know it. I was always using place kittens ;) And for text 3 different lorem ipsum generators :dash: Nice!